package.json

```diff
@@ -20,12 +20,11 @@
     "build:storybook:stats": "pnpm turbo build:storybook -- --webpack-stats-json",
     "test:unit": "turbo run test:unit",
     "test:unit:coverage": "turbo run test:unit:coverage",
-    "test:e2e": "playwright test -c e2e",
-    "test:e2e:headed": "pnpm run test:e2e --headed",
-    "test:e2e:debug": "PWDEBUG=1 pnpm run test:e2e",
+    "test:e2e": "turbo e2e --filter @ultraviolet/e2e",
     "check:deps": "npx depcheck . --skip-missing=true --ignores='bin,eslint,vite,jest,husky,@commitlint/*,@babel/*,babel-*'",
     "commit": "npx git-cz -a --disable-emoji",
     "start": "STORYBOOK_ENVIRONMENT=development storybook dev -p 6006",
+    "start:e2e": "turbo start --filter @ultraviolet/e2e",
```

Comment:
needed at the root level of the package ?

Reply:
not really it's a legacy tryout